{"timestamp":"2025-10-14T02:59:49.190371","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-10-14T02:59:49.191169","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/deploy_monitor_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-10-14T02:59:49.349519Z","level":"info","event":"Running: python3 /opt/airflow/project/DEPLOY/build_docker_image.py --no-build","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.349594Z","level":"info","event":"","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.349619Z","level":"info","event":"python3: can't open file '/opt/airflow/project/DEPLOY/build_docker_image.py': [Errno 2] No such file or directory","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.349639Z","level":"info","event":"","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.349657Z","level":"info","event":"Running: python3 /opt/airflow/project/DEPLOY/build_docker_image.py","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.349674Z","level":"info","event":"","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.349690Z","level":"info","event":"python3: can't open file '/opt/airflow/project/DEPLOY/build_docker_image.py': [Errno 2] No such file or directory","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.349705Z","level":"info","event":"","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-14T02:59:49.251671","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"RuntimeError","exc_value":"Script failed: python3 /opt/airflow/project/DEPLOY/build_docker_image.py\npython3: can't open file '/opt/airflow/project/DEPLOY/build_docker_image.py': [Errno 2] No such file or directory\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":838,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1130,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":212,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":235,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/deploy_monitor_dag.py","lineno":55,"name":"deploy_py"},{"filename":"/opt/airflow/dags/deploy_monitor_dag.py","lineno":36,"name":"run_script"}]},{"exc_type":"RuntimeError","exc_value":"Script failed: python3 /opt/airflow/project/DEPLOY/build_docker_image.py --no-build\npython3: can't open file '/opt/airflow/project/DEPLOY/build_docker_image.py': [Errno 2] No such file or directory\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/opt/airflow/dags/deploy_monitor_dag.py","lineno":53,"name":"deploy_py"},{"filename":"/opt/airflow/dags/deploy_monitor_dag.py","lineno":36,"name":"run_script"}]}]}
